coaster-nn/src/frameworks/native/helper.rs

```diff
@@ -4,15 +4,19 @@ use crate::co;
 use crate::co::plugin::Error as PluginError;
 use crate::co::plugin::numeric_helpers::Float;
 use crate::co::frameworks::native::flatbox::FlatBox;
+use ::{DirectionMode, RnnInputMode};
```

Comment:
We should consider using `crate::` here instead of `::`, what does clipppy say about this?

Reply:
It didn't complain, oddly. 

Did find a bug in #81 though, which is really weird because it passed CI. CI doesn't appear to be catching a lot of build errors right now?

Reply:
I am going let this hang for two more days, until I fixed the CI up.
There are few simple things I can improve rather quickly, https://github.com/spearow/juice/issues/86